src/app/compiler.js

```diff
@@ -189,7 +200,9 @@ function Compiler (editor, queryParams, handleGithubCall, updateFiles) {
       compilationFinished({ error: 'Worker error: ' + msg.data });
     });
     compileJSON = function (source, optimize) {
-      worker.postMessage({cmd: 'compile', source: JSON.stringify(source), optimize: optimize});
+      jobs[nextJobId] = {sources: source};
```

Comment:
No, it is not an array. Not sure how javascript implements arrays, but since we usually only have one concurrent job, I think an object is better.